instrumentation-api-incubator/src/test/java/io/opentelemetry/instrumentation/api/incubator/semconv/db/DbClientAttributesExtractorTest.java

```diff
@@ -90,9 +89,9 @@ void shouldExtractAllAvailableAttributes() {
               entry(DbIncubatingAttributes.DB_CONNECTION_STRING, "mydb:///potatoes"),
               entry(DbIncubatingAttributes.DB_STATEMENT, "SELECT * FROM potato"),
               entry(DbIncubatingAttributes.DB_OPERATION, "SELECT"),
-              entry(AttributeKey.stringKey("db.namespace"), "potatoes"),
-              entry(AttributeKey.stringKey("db.query.text"), "SELECT * FROM potato"),
-              entry(AttributeKey.stringKey("db.operation.name"), "SELECT"));
+              entry(DbIncubatingAttributes.DB_NAMESPACE, "potatoes"),
```

Comment:
is there any reason not to use static imports for all of these?

Reply:
This is only to minimize the diff and keep things consistent, do you think we should migrate and use static imports like in some other files ?

Reply:
I think we could skip static imports in this file to reduce diff size. In https://github.com/open-telemetry/opentelemetry-java-instrumentation/pull/12920 @trask used a script to automatically add the static imports, but that script only ran under instrumentation. We could run that script in `instrumentation-api-incubator` in a separate PR. Use static imports in the changes that are under instrumentation, all (or at least most) of the usages should already use static imports there.